lib/sycamore/sycamore/tests/integration/connectors/opensearch/test_opensearch_read.py

```diff
@@ -81,6 +91,13 @@ def test_ingest_and_read(self, setup_index, exec_mode):
             .take_all()
         )
 
+        os_client.indices.refresh(TestOpenSearchRead.INDEX)
+
+        expected_count = len(original_docs)
+        actual_count = get_doc_count(os_client, TestOpenSearchRead.INDEX)
+        # refresh should have made all ingested docs immediately available for search
+        assert actual_count == expected_count, f"Expected {expected_count} documents, found {actual_count}"
```

Comment:
FYI: Golang taught me to use got & want, which are shorter and still clear.  Fine to keep as is.